addon/components/paper-menu-item.js

```diff
@@ -11,6 +11,9 @@ export default Component.extend({
       this.nearestOfType(PaperMenuAbstract).send('toggleMenu');
       this.sendAction('onClick', event);
     }
+  },
+  mouseEnter() {
+  	this.$('.md-button:not([disabled])').focus();
```

Comment:
A test is failing because the indentation is incorrect here.
You can run `ember test` to make sure the tests pass before commits.